examples/hello-world/step-by-step/cifar10/stats/image_stats.ipynb

```diff
@@ -37,7 +37,7 @@
     "\n",
     "## Setup NVFLARE\n",
     "\n",
-    "Follow [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html) to set up a virtual environment and install NVFLARE."
+    "Follow [Getting Started](../../../../getting_started/readme.ipynb) to set up a virtual environment and install NVFLARE.\n"
```

Comment:
we need to update our getting started pages later, right now the newest getting started does not have installation instructions